### PR TITLE
feat(policy): improve meshmetric docs for opentelemetry backend

### PR DESCRIPTION
**Pull request by Automaat**
Fixes: #1676 

Removed MeshMetric OpenTelemtry limitations from docs, and added description on how to push metrics from app to OpenTelemetry collector

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits)

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: app/_src/policies/meshmetric.md [hunk 1]
```diff
@@ -101,6 +101,13 @@ applications:
     port: 8888 # port on which application is listening
 ```
 
+{% if_version gte:2.7.x %}
+{% warning %}
+Metrics exposed in application needs to be in Prometheus format in order for Dataplane Proxy to parse it and expose. Either 
+for Prometheus or OpenTelemetry backend.
```
Comment (slonka) [inline app/_src/policies/meshmetric.md:107]:
```suggestion
Metrics exposed by the application need to be in Prometheus format for the Dataplane Proxy to be able to parse and expose them to either Prometheus or OpenTelemetry backend.
```

File: app/_src/policies/meshmetric.md [hunk 2]
```diff
@@ -275,6 +283,62 @@ When you configure application scraping make sure to specify `application.name`
...
+{% if_version gte:2.7.x %}
+```yaml
+backends:
+  - type: OpenTelemetry
+    openTelemetry: 
+      endpoint: http://otel-collector.observability.svc:4317
+      refreshInterval: 60s
+```
+
+This configuration tells {{site.mesh_product_name}} data plane proxy to push metrics to [OpenTelemetry collector](https://opentelemetry.io/docs/collector/).
```
Comment (slonka) [inline app/_src/policies/meshmetric.md:297]: What's the correct capitalisation / spelling of dataplane proxy? We have https://github.com/search?q=repo%3Akumahq%2Fkuma-website%20%22dataplane%20proxy%22&type=code and https://github.com/search?q=repo%3Akumahq%2Fkuma-website%20%22dataplane%20proxy%22&type=code

The previous paragraph uses this:

```suggestion
This configuration tells {{site.mesh_product_name}} Dataplane Proxy to push metrics to [OpenTelemetry collector](https://opentelemetry.io/docs/collector/).
```

File: app/_src/policies/meshmetric.md [hunk 3]
```diff
@@ -101,6 +101,13 @@ applications:
     port: 8888 # port on which application is listening
 ```
 
+{% if_version gte:2.7.x %}
+{% warning %}
```
Comment (slonka) [inline app/_src/policies/meshmetric.md:105]: should we put this at the top of the paragraph?

**Comment by slonka** [comment]
Rendered: https://deploy-preview-1679--kuma.netlify.app/docs/dev/policies/meshmetric/#meshmetric

File: app/_src/policies/meshmetric.md [hunk 4]
```diff
@@ -275,6 +283,62 @@ When you configure application scraping make sure to specify `application.name`
...
+```yaml
+backends:
+  - type: OpenTelemetry
+    openTelemetry: 
+      endpoint: http://otel-collector.observability.svc:4317
+      refreshInterval: 60s
+```
+
+This configuration tells {{site.mesh_product_name}} data plane proxy to push metrics to [OpenTelemetry collector](https://opentelemetry.io/docs/collector/).
+Dataplane Proxy will scrape metrics from Envoy and other [applications](/docs/{{ page.version }}/policies/meshmetric/#applications) in a Pod/VM.
```
Comment (slonka) [inline app/_src/policies/meshmetric.md:298]: I think unwanted period:

```suggestion
Dataplane Proxy will scrape metrics from Envoy and other [applications](/docs/{{ page.version }}/policies/meshmetric/#applications) in a Pod/VM
```

File: app/_src/policies/meshmetric.md [hunk 5]
```diff
@@ -275,6 +283,62 @@ When you configure application scraping make sure to specify `application.name`
...
+and push them to configured OpenTelemetry collector, every **60 seconds**.
```
Comment (slonka) [inline app/_src/policies/meshmetric.md:299]: that's only default, can be changed - so maybe something like this:

```suggestion
and push them to configured OpenTelemetry collector, by default every **60 seconds** (use `refreshInterval` to change it).
```

File: app/_src/policies/meshmetric.md [hunk 6]
```diff
@@ -275,6 +283,62 @@ When you configure application scraping make sure to specify `application.name`
...
+
+When you configure application scraping make sure to specify `application.name` to utilize [OpenTelemetry scoping](https://opentelemetry.io/docs/concepts/instrumentation-scope/)
+
+#### Pushing metrics from application to OpenTelemetry collector directly
+
+Right now if you want to expose metrics from you application to OpenTelemetry collector you can access collector directly.
```
Comment (slonka) [inline app/_src/policies/meshmetric.md:305]:
```suggestion
Right now if you want to expose metrics from your application to OpenTelemetry collector you can access collector directly.
```

File: app/_src/policies/meshmetric.md [hunk 7]
```diff
@@ -275,6 +283,62 @@ When you configure application scraping make sure to specify `application.name`
...
+    openTelemetry: 
+      endpoint: http://otel-collector.observability.svc:4317
+      refreshInterval: 60s
+```
+
+This configuration tells {{site.mesh_product_name}} data plane proxy to push metrics to [OpenTelemetry collector](https://opentelemetry.io/docs/collector/).
+Dataplane Proxy will scrape metrics from Envoy and other [applications](/docs/{{ page.version }}/policies/meshmetric/#applications) in a Pod/VM.
+and push them to configured OpenTelemetry collector, every **60 seconds**.
+
+When you configure application scraping make sure to specify `application.name` to utilize [OpenTelemetry scoping](https://opentelemetry.io/docs/concepts/instrumentation-scope/)
```
Comment (slonka) [inline app/_src/policies/meshmetric.md:301]:
```suggestion
When you configure application scraping make sure to specify `application.name` to utilize [OpenTelemetry scoping](https://opentelemetry.io/docs/concepts/instrumentation-scope/).
```

**Review by slonka — COMMENTED** [review]
Some nitpicks

File: app/_src/policies/meshmetric.md (hunk 3 above)
Comment (Automaat) [reply app/_src/policies/meshmetric.md]: moved

**Review by slonka — APPROVED** [review]

**Pull request merged**